### ISSUE TITLE
Podcast Header Images - Use Empty Alt Text

### ISSUE THREAD

**Issue by ryanmccombe** [bbc/simorgh#8910]
**Is your feature request related to a problem? Please describe.**
The alt text for the image in podcast pages is the default alt text for the service

We've been asked to remove this for a11y UX, as the current string is unhelpful

**Describe the solution you'd like**
Set a blank alt text for that image, on podcasts only.  Radio episodes should still have the existing implementation

**Context/discussion**
https://bbcnews.slack.com/archives/CK85Q5SL9/p1614005259001200

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
